src/controllers/users.controller.js

```diff
@@ -0,0 +1,97 @@
+const { userSchema } = require('../schemas');
+const { dynamicSchema } = require('../utils');
+const { usersService } = require('../services');
+
+const getAll = async (req, res) => {
+  const users = await usersService.getAll();
+
+  res.json(users);
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await usersService.getById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.json(user);
+};
+
+const create = async (req, res) => {
+  const { error, value } = userSchema.validate(req.body, {
+    allowUnknown: true,
+  });
+
+  if (error) {
+    res.sendStatus(400);
```

Comment:
Consider providing a more descriptive error message when validation fails, such as `res.status(400).json({ error: error.details[0].message });` to help the client understand what went wrong.